modules/tinymce/src/core/test/ts/browser/fmt/FormatChangeVarsTest.ts

```diff
@@ -0,0 +1,194 @@
+import { Waiter } from '@ephox/agar';
+import { after, before, beforeEach, context, describe, it } from '@ephox/bedrock-client';
+import { Type } from '@ephox/katamari';
+import { TinyHooks, TinySelections } from '@ephox/mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.fmt.FormatChangeVarsTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce'
+  }, []);
```

Comment:
You've forgotten to import and pass the `Theme` here btw.

Reply:
Thanks, fixed now